index.html

```diff
@@ -0,0 +1,59 @@
+<DOCTYPE html>
+<html>
+
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Mark Marchant - Portfolio</title>
+        <link rel="stylesheet" href="./css/main.css">
+    </head>
+    <body>
+            <nav class="nav">
+                <ul class="nav__menu">
+                    <li><a href="index.html">HOME</a></li>
+                    <li><a href="#">ABOUT ME</a></li>
+                    <li><a href="#">PROJECTS</a></li>
+                    <li><a href="#">CONTACT ME</a></li>
+                </ul>
+            </nav>
+            <div class="intro">
+                <h2>Mark Philip Marchant</h2>
+                <h2>Software Developer</h2>
+            </div>
+        </div>
+        <div class="section__header">Welcome</div><hr> 
+        <div class="section section__welcome">                          
+            <img src="./img/Mark1.jpg" />
+            <p>
+                Welcome to my portfolio. I very new to programming and I learn day by day. Not much to look at now but I will do my best to add projects.
+            </p>
+        </div>
+            <div class="section section__about">
```

Comment:
The double underscores in "section__about" indicate that it should be a child of "section". You're likely looking for a modifier. When naming the modifier, try to think about how the styling is different from a basic section. For example, "section--largeText", "section--mediumText", "section--smallText".